opel-collector/values.yaml

```diff
@@ -0,0 +1,95 @@
+opentelemetry-collector:
+  mode: daemonset
+
+  fullnameOverride: otel-agent
+
+  extraEnvs:
+  - name: CORALOGIX_PRIVATE_KEY
+    valueFrom:
+      secretKeyRef:
+        name: coralogix-otel-privatekey
+        key: PRIVATE_KEY
+  - name: APP_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+  - name: SUBSYSTEM
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+  - name: KUBE_NODE_NAME
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: spec.nodeName
+  - name: CORALOGIX_ENDPOINT
+    value: "tracing-ingress.eu2.coralogix.com"
```

Comment:
I can change it but I think there should be a default endpoint that can be override, but if I take this env down, the whole deploy will fail since the exporter wont find the endpoint env